userprefs.js

```diff
@@ -0,0 +1,14 @@
+window.addEventListener("DOMContentLoaded", () => {
```

Comment:
Let's move this to `scripts/` so it's not hanging out here in the root. `userprefs.php` probably shouldn't be here either but that ship has sailed.